plugins/Jobs.cpp

```diff
@@ -1005,11 +1006,22 @@ void BedrockJobsCommand::process(SQLite& db) {
             return;
         }
 
-        // If this is RetryJob and we want to update the name, let's do that
+        // If this is RetryJob and we want to update the name and/or priority, let's do that
         const string& name = request["name"];
-        if (!name.empty() && SIEquals(requestVerb, "RetryJob")) {
-            if (!db.writeIdempotent("UPDATE jobs SET name=" + SQ(name) + " WHERE jobID=" + SQ(jobID) + ";")) {
-                STHROW("502 Failed to update job name");
+        if (SIEquals(requestVerb, "RetryJob")) {
+            list<string> updates;
+            if (!name.empty()) {
+                updates.push_back("name=" + SQ(name) + " ");
+            }
+            if (request.isSet("jobPriority")) {
+                _validatePriority(request.calc64("jobPriority"));
+                updates.push_back("priority=" + SQ(request["jobPriority"]) + " ");
+            }
+            if (!updates.empty()) {
```

Comment:
No throw if the user try to run the command with neither name or priority? The query will fail later if you don't do that.

Reply:
Nvm, it's Friday and I don't know how to read.